doc/changes/changes-20200803.md

```diff
@@ -0,0 +1,48 @@
+  * [script-languages-release@#60](https://github.com/exasol/script-languages-release/pull/60): Handle multiple lines at once in LogHandlers in exaslct to fix some bugs on MacOSx (but MacOSx is still highly experimental)
+ 
+## Features / Enhancements
+ 
+* UDFs:
+  * [script-languages-release@#49](https://github.com/exasol/script-languages-release/pull/49): Introduce error codes into UDFClient
+  * [script-languages-release@#50](https://github.com/exasol/script-languages-release/pull/50): Complete, but compact stacktraces for Java and Python UDFs, Java stacktraces with causes and introduce error codes for Java, Python and R UDFs)
+  * [script-languages-release@#54](https://github.com/exasol/script-languages-release/pull/54): Support for chained exception for Python 3 UDFs
+  * [script-languages-release@#40](https://github.com/exasol/script-languages-release/pull/40): Improve shutdown handling of the UDFClient to reduce the risk of orphaned exaudfclient processes in error cases
+* Script-Language Containers:
+  * [script-languages-release@#47](https://github.com/exasol/script-languages-release/pull/47): Add seperate standard-flavor for Exasol 7.0 with pinned package versions
+  * [script-languages-release@#38](https://github.com/exasol/script-languages-release/pull/38): Cleanup standard-flavor for Exasol 6.1+ to reduce the container size
+  * [script-languages-release@#26](https://github.com/exasol/script-languages-release/issues/26): Add boto3 to standard flavor 
+  * [script-languages-release@#25](https://github.com/exasol/script-languages-release/issues/25): Add pysftp and paramiko to standard flavor
+  * [script-languages-release@#32](https://github.com/exasol/script-languages-release/issues/32): Add exasol websocket-api for Python3 to standard flavor
+  * [script-languages-release@#57](https://github.com/exasol/script-languages-release/issues/57): Simplify the customization of flavors for users
+  * [script-languages-release@#23](https://github.com/exasol/script-languages-release/issues/23): Add script to install specific version of a R packages
+  * [script-languages@#127](https://github.com/exasol/script-languages/pull/127): Add more robust package install scripts which support package versions
+* Exslsct:
+  * [script-languages-release@#42](https://github.com/exasol/script-languages-release/pull/42): Add command to generate language activation statement
+
+## Refactorings
+
+* Exaslct:
+  * [script-languages-release@#46](https://github.com/exasol/script-languages-release/pull/46): Reorganizing exaslct and extracting the test environment to [exasol/integration-test-docker-environment](https://github.com/exasol/integration-test-docker-environment)
+  * [script-languages@#108](https://github.com/exasol/script-languages/pull/108): Add cleanup method to exaslct tasks and scripts to run exaslct tests
+  * [script-languages@#98](https://github.com/exasol/script-languages/pull/98): Pass docker socket to test container
```

Comment:
```suggestion
  * [script-languages@#98](https://github.com/exasol/script-languages/pull/98): Pass a docker socket to the test container
```

Reply:
`Pass the docker socket to the test container`